ember-file-upload/package.json

```diff
@@ -63,12 +63,16 @@
     "@babel/plugin-transform-runtime": "^7.18.2",
     "@babel/plugin-transform-typescript": "^7.18.4",
     "@babel/preset-typescript": "^7.17.12",
+    "@babel/runtime": "^7.20.7",
+    "@ember/string": "^3.0.1",
```

Comment:
This may need to go into `peerDependency` to avoid a deprecation in Ember 4.10+. Adding as peer dependency would be a breaking change. Maybe better done in separate PR.

When doing a new major for that, we should also revisit other dependencies and see if they should be peer dependencies instead. E.g. `@glimmer/tracking` and `@glimmer/component`.

Reply:
Ah good point. Definitely better done in a separate PR!

Reply:
Added a tracking issue #899